NEWS

```diff
@@ -93,6 +93,7 @@
 * Fixed Active Message hanging issue in ucp_client_server
 * Fixed potential crash due to redundant munmap call in ucp mmap tests
 * Fixed a crash when running CUDA gtest under valgrind
+* Fixed endpoint timeout issue by increasing UD timeout under Valgrind
```

Comment:
Maybe:

Fixed UD endpoint timeout issue under Valgrind

(focus more on the issue fixed, rather than how it was done).

Reply:
\+ more concise